bundles/org.openhab.binding.cul/src/main/resources/OH-INF/thing/cul_bridge.xml

```diff
@@ -0,0 +1,77 @@
+<?xml version="1.0" encoding="UTF-8"?>
+<thing:thing-descriptions bindingId="maxcul"
+	xmlns:xsi="http://www.w3.org/2001/XMLSchema-instance"
+	xmlns:thing="https://openhab.org/schemas/thing-description/v1.0.0"
+	xsi:schemaLocation="https://openhab.org/schemas/thing-description/v1.0.0 https://openhab.org/schemas/thing-description-1.0.0.xsd">
+
+	<bridge-type id="cul_max_bridge">
```

Comment:
This binding has a similar bridge style as the modbus binding. That binding uses the name `serial` and `tcp` for the bridge names. I would suggest to use the same naming for the bridges.

Reply:
CUL and CUN are the common terms for this device (see http://culfw.de/culfw.html, https://shop.busware.de/product_info.php/products_id/44 and https://github.com/heliflieger/a-culfw/tree/master/culfw).  Not entirely sure what the abbreviation stands for, but I'd suggest sticking with these for clarity.